app/models/miq_server.rb

```diff
@@ -590,11 +590,23 @@ def self.zone_is_modifiable?
     Zone.visible.in_my_region.count > 1
   end
 
+  def self.managed_providers
+    providers = {}
+    ExtManagementSystem.all.each { |ems| providers[ems.type] = ems.class.count }
+
+    providers
+  end
+
   def self.managed_resources
     {
       :vms                     => Vm.active.count,
       :hosts                   => Host.active.count,
       :aggregate_physical_cpus => Host.active.in_my_region.sum(:aggregate_physical_cpus),
+      :providers               => managed_providers,
+      :deployment              => MiqEnvironment::Command.is_appliance? ? "appliance" : "podified",
+      :arch                    => MiqEnvironment::Command.arch.to_s,
```

Comment:
Now that we are adding arch and other details, I'm wondering if the method name makes sense anymore.  I'm thinking we should rename this method to `audit_details` or something like that, and then rename `audit_managed_resources` to `report_audit_details`.

Then, of course you'll have to find the callers (which I think should only be the scheduler, and of course what we have downstream).

Reply:
Additionally, I don't think we are using unmanaged_resources method anymore so we can delete that if we aren't.

Reply:
It looks like we still use `unmanaged_resources` here https://github.com/ManageIQ/manageiq/blob/master/app/models/miq_server/at_startup.rb#L65? Which looks like is called here https://github.com/ManageIQ/manageiq/blob/master/lib/workers/evm_server.rb#L130

Reply:
Oh interesting - I forgot we had that.  I wonder if we should just dump the entirety of the audit_details payload there instead.